polkadotjs-examples/src/utils.ts

```diff
@@ -50,7 +50,7 @@ export const getAddressId = (blockchain: PalletCreditcoinBlockchain | string, ex
 };
 
 export const randomEthAddress = () => {
-    const id = randomBytes(32).toString('hex');
+    const id = randomBytes(20).toString('hex').toLowerCase();
```

Comment:
Shouldn't this be 32 bytes? It is for a private key. 
https://docs.ethers.io/v5/api/signer/#Wallet
Even better their is `Wallet.createRandom()` function.